.circleci/config.yml

```diff
@@ -154,6 +156,20 @@ jobs:
       - run: yarn install --cwd=dashboard --frozen-lockfile
       - run: yarn --cwd=dashboard run lint
       - run: yarn --cwd=dashboard run test --maxWorkers=4 --coverage
+  test_chart:
+    docker:
+      - image: circleci/golang:1.9
+    environment:
+      HELM_VERSION: v2.9.1
+    steps:
+      - checkout
+      - run: |
+          wget https://storage.googleapis.com/kubernetes-helm/helm-$HELM_VERSION-linux-amd64.tar.gz
+          tar zxf helm-$HELM_VERSION-linux-amd64.tar.gz
+          sudo mv linux-amd64/helm /usr/local/bin/
+          helm init --client-only
+      - run: helm dep build ./chart/kubeapps/
+      - run: helm template ./chart/kubeapps/
```

Comment:
Ideally I'd like to add some combinations related to the tls configuration.

All these should pass

```
helm template --set tillerProxy.tls.key="my-key" --set tillerProxy.tls.cert="my-cert"  .
helm template --set tillerProxy.tls.verify=false --set tillerProxy.tls.key="my-key" --set tillerProxy.tls.cert="my-cert"  .
 helm template --set tillerProxy.tls.verify=true --set tillerProxy.tls.key="my-key" --set tillerProxy.tls.cert="my-cert" --set tillerProxy.tls.ca="my-ca" .
```

Does it make sense? Should I add it in another patch?

Reply:
We can add several combinations but I would do that in a script to be easier to execute locally or migrate if needed. I don't think we need a different patch for that, we can do that here.